ios/RNDeviceInfo/RNDeviceInfo.m

```diff
@@ -375,6 +376,27 @@ - (BOOL) isTablet {
     }
 }
 
+RCT_EXPORT_METHOD(getDeviceToken:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject) {
+    if (@available(iOS 11.0, *)) {
```

Comment:
Nice - may I assume you've tested this on ios < 11? I have ios 9 available as a real device still and of course with simulators, just curious.

Also, I am not super strong on Xcode, is the '@available' syntax limited to a specific Xcode version? I have a small memory that this might be Xcode 11+ only?

To be clear, I am 100% okay with breaking changes, I just follow semver strictly when I do them so I want to be careful + clear :-)

Reply:
I've tested this on iOS 10.3 :)

Regarding the `@available` syntax, I didn't think about it but it was introduced in Xcode 9, so I guess it's ok:

![image](https://user-images.githubusercontent.com/17070498/70530875-1b73d400-1b54-11ea-9e65-4fa3c9c40327.png)
(source [Xcode 9 Release notes](https://developer.apple.com/library/archive/releasenotes/DeveloperTools/RN-Xcode/Chapters/Introduction.html#//apple_ref/doc/uid/TP40001051-CH1-SW878))

Reply:
fantastic! thanks for double-checking. I don't see any semver concern then I'll get this packed up and released as soon as I get a chance - until then you can confidently use patch-package to integrate it with the knowledge the next upstream release will contain the identical change. Thanks again

Reply:
or I guess now that it's committed you can depend on master and the commit hash if that's easier :-)